Gopkg.lock

```diff
@@ -336,18 +378,18 @@
   revision = "792786c7400a136282c1664665ae0a8db921c6c2"
   version = "v1.0.0"
 
-[[projects]]
-  branch = "master"
-  name = "github.com/pquerna/cachecontrol"
-  packages = [".","cacheobject"]
-  revision = "5475d973ea70916980bee28c2b674f3dc3eaed0a"
-
 [[projects]]
   name = "github.com/satori/go.uuid"
   packages = ["."]
   revision = "879c5887cd475cd7864858769793b2ceb0d44feb"
   version = "v1.1.0"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/seanhuxy/libkv"
```

Comment:
I see you added your fork as a dependency. I'm totally ok with reusing some of our code from external repos, but since you replaced the original repo with your fork, you would have to maintain your fork and pull patches, resolve merge conflicts, etc. Based on my brief research on docker/libkv, I doubt they will be interested in a PR for postgres support.

I would suggest implementing it in dispatch entitystore instead. we already provide an abstraction, we just need different implementation based on postgres. this way you won't need to use libkv at all, and you can put your code directly in vmware/dispatch.

Reply:
This was my idea.  I see your point @kars7e.  I haven't researched docker/libkv deeply.  I'm not sure why they wouldn't accept a Postgres backend, but since we do have our own abstraction, I guess it could work either way.  I don't have a really strong opinion on this.

Reply:
@kars7e 's opinion sounds good to me, but we may (1) implement another identical libkv in our own repo OR (2) lose the flexibility of switching backend db. I understand my version of libkv is not perfect (has bugs), but I suggest that we should keep it and make things work and simple, extensible first. I am also ok with and willing to move it to a vmware controlled repo (a vmware forked libkv)

Reply:
Well, I have a feeling in the medium term we may migrate from libkv.  Certainly if we use postgres, there will likely come a time where we actually want to use some of it's features.  I think @kars7e is right in that we already have a defined interface (entity store).  We can code to that, and in the near term we still have all of the advantages.  I'm kind of siding with @kars7e.